File: 2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.java
class Solution {
    public int[] leftRightDifference(int[] nums) {
        int[] answer = new int[nums.length];
        int total = Arrays.stream(nums).sum();
        int running_sum = 0;

        for (int i=0; i < nums.length; i++){
            answer[i] = Math.abs(2*running_sum - total + nums[i]);
            running_sum += nums[i];
        }

        return answer;
    }
}

File: 2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.py
class Solution:
    def leftRightDifference(self, nums: List[int]) -> List[int]:
        answer = []
        total = sum(nums)
        running_sum = 0

        for num in nums:
            answer.append(abs(2*running_sum - total + num))
            running_sum += num

        return answer
